tendrl/node_agent/manager/rpc.py

```diff
@@ -0,0 +1,134 @@
+import json
+import logging
+import re
+import traceback
+import uuid
+
+import etcd
+import gevent.event
+
+from tendrl.node_agent.config import TendrlConfig
+from tendrl.node_agent.flows.flow_execution_exception import \
+    FlowExecutionFailedError
+
+
+config = TendrlConfig()
+LOG = logging.getLogger(__name__)
+
+
+class EtcdRPC(object):
+
+    def __init__(self):
+        etcd_kwargs = {'port': int(config.get("bridge_common", "etcd_port")),
+                       'host': config.get("bridge_common", "etcd_connection")}
+
+        self.client = etcd.Client(**etcd_kwargs)
+        self.bridge_id = str(uuid.uuid4())
+
+    def _process_job(self, raw_job, job_key):
+        # Pick up the "new" job that is not locked by any other bridge
+        if raw_job['status'] == "new" and raw_job["sds_type"] == "generic":
```

Comment:
I think we need to separate "sds" jobs and "node" jobs, basically have a raw_job["type"] = sds|node
Please communicate this to @anivargi